pkg/ddevapp/ddevapp.go

```diff
@@ -2567,45 +2567,45 @@ func (app *DdevApp) DockerEnv() map[string]string {
 		"DDEV_DATABASE":                  app.Database.Type + ":" + app.Database.Version,
 		"DDEV_FILES_DIR":                 app.GetContainerUploadDir(),
 		"DDEV_FILES_DIRS":                strings.Join(app.GetContainerUploadDirs(), ","),
-
-		"DDEV_HOST_DB_PORT":             dbPortStr,
-		"DDEV_HOST_MAILHOG_PORT":        app.HostMailpitPort,
-		"DDEV_HOST_MAILPIT_PORT":        app.HostMailpitPort,
-		"DDEV_HOST_HTTP_PORT":           hostHTTPPortStr,
-		"DDEV_HOST_HTTPS_PORT":          hostHTTPSPortStr,
-		"DDEV_HOST_WEBSERVER_PORT":      hostHTTPPortStr,
-		"DDEV_MAILHOG_HTTPS_PORT":       app.GetMailpitHTTPSPort(),
-		"DDEV_MAILHOG_PORT":             app.GetMailpitHTTPPort(),
-		"DDEV_MAILPIT_HTTP_PORT":        app.GetMailpitHTTPPort(),
-		"DDEV_MAILPIT_HTTPS_PORT":       app.GetMailpitHTTPSPort(),
-		"DDEV_MAILPIT_PORT":             app.GetMailpitHTTPPort(),
-		"DDEV_XHGUI_HTTP_PORT":          app.GetXHGuiHTTPPort(),
-		"DDEV_XHGUI_HTTPS_PORT":         app.GetXHGuiHTTPSPort(),
-		"DDEV_DOCROOT":                  app.GetDocroot(),
-		"DDEV_HOSTNAME":                 app.HostName(),
-		"DDEV_UID":                      uidStr,
-		"DDEV_GID":                      gidStr,
-		"DDEV_USER":                     username,
-		"DDEV_MUTAGEN_ENABLED":          strconv.FormatBool(app.IsMutagenEnabled()),
-		"DDEV_PHP_VERSION":              app.PHPVersion,
-		"DDEV_WEBSERVER_TYPE":           app.WebserverType,
-		"DDEV_PROJECT_TYPE":             app.Type,
-		"DDEV_ROUTER_HTTP_PORT":         app.GetPrimaryRouterHTTPPort(),
-		"DDEV_ROUTER_HTTPS_PORT":        app.GetPrimaryRouterHTTPSPort(),
-		"DDEV_XDEBUG_ENABLED":           strconv.FormatBool(app.XdebugEnabled),
-		"DDEV_XHPROF_MODE":              app.GetXHProfMode(),
-		"DDEV_PRIMARY_URL":              primaryURL,
-		"DDEV_PRIMARY_URL_PORT":         primaryURLPort,
-		"DDEV_PRIMARY_URL_WITHOUT_PORT": primaryURLWithoutPort,
-		"DDEV_SCHEME":                   scheme,
-		"DDEV_VERSION":                  versionconstants.DdevVersion,
-		"DOCKER_SCAN_SUGGEST":           "false",
-		"DDEV_GOOS":                     runtime.GOOS,
-		"DDEV_GOARCH":                   runtime.GOARCH,
-		"IS_DDEV_PROJECT":               "true",
-		"IS_GITPOD":                     strconv.FormatBool(nodeps.IsGitpod()),
-		"IS_CODESPACES":                 strconv.FormatBool(nodeps.IsCodespaces()),
-		"IS_WSL2":                       isWSL2,
+		"DDEV_GLOBAL_DIR":                util.WindowsPathToCygwinPath(globalconfig.GetGlobalDdevDir()),
```

Comment:
I wonder if this transformation should be done in a function. We're not having to do this anywhere else?